soem: 1.3.0-0 in 'kinetic/distribution.yaml' [bloom]

smits opened the pull request.

dirk-thomas commented on kinetic/distribution.yaml:
```diff
@@ -7718,6 +7718,22 @@ repositories:
+      url: https://github.com/smits/soem-gbp.git
+      version: 1.3.0-0
+    source:
+      test_pull_requests: true
```
Just checking: did you follow the instructions to provide the necessary access for for the @ros-pull-request-builder user?

smits replied:
Where do I find these instructions, I just ran the bloom-release process.

smits replied:
I've followed the instructions on http://wiki.ros.org/buildfarm/Pull%20request%20testing, I've manually added the webhook so it should be good to go.

dirk-thomas replied:
Did you already grant write access to the repo to the @ros-pull-request-builder user (as mentioned in the instructions)?

dirk-thomas commented:
> Did you already grant write access to the repo to the @ros-pull-request-builder user (as mentioned in the instructions)?

@smits Can you please respond to this earlier comment since otherwise the PR integration won't work.

@tfoote It would be better to wait for a response before merging since that keep the responsibility to act with the author of the PR. Now we have to follow up on this topic if there is no response to remove the flag again. Please do that within the next in case we don't hear back on this.

tfoote commented:
> @tfoote It would be better to wait for a response before merging since that keep the responsibility to act with the author of the PR. Now we have to follow up on this topic if there is no response to remove the flag again. Please do that within the next in case we don't hear back on this.

Merging this doesn't transfer the responsibility for this. It is still the responsibility of the maintainer to validate that the PR is working as stated.

From the above comments @smits stated that he followed the instructions so I am taking the basic assumption that he has completed adding the write access. If you, who receive ros-pull-request-builders notifications, mentioned that you hadn't seen a request to grant access I would believe otherwise. But you didn't mention anything like that or otherwise indicate that you didn't think that ros-pull-request-builder had access in your comment.

Thus without any statement to the otherwise I took the assumption that it was complete. Looking into it using the ros-pull-request-builder It appears that @smits has not added write access. And in reviewing the instructions now I saw that adding Write access was buried in the paragraph header, and not in the enumerated tasks in the "alternate" approach. I've [updated](http://wiki.ros.org/action/diff/buildfarm/Pull%20request%20testing?action=diff&rev1=18&rev2=19) the docs to make that clearer as I believe that it was not clearly enough documented if an experienced community member missed a step.

smits commented:
Hi Tully, Dirk,

Sorry to not have responded earlier, I was indeed under the impression that
following the manual procedure on the wiki did not require me to add write
access to @ros-pull-request-builder.  I've added him as collaborator,
you've should have received an invitation.

Op zo 9 jul. 2017 09:08 schreef Tully Foote <notifications@github.com>:

> @tfoote <https://github.com/tfoote> It would be better to wait for a
> response before merging since that keep the responsibility to act with the
> author of the PR. Now we have to follow up on this topic if there is no
> response to remove the flag again. Please do that within the next in case
> we don't hear back on this.
>
> Merging this doesn't transfer the responsibility for this. It is still the
> responsibility of the maintainer to validate that the PR is working as
> stated.
>
> From the above comments @smits <https://github.com/smits> stated that he
> followed the instructions so I am taking the basic assumption that he has
> completed adding the write access. If you, who receive
> ros-pull-request-builders notifications, mentioned that you hadn't seen a
> request to grant access I would believe otherwise. But you didn't mention
> anything like that or otherwise indicate that you didn't think that
> ros-pull-request-builder had access in your comment.
>
> Thus without any statement to the otherwise I took the assumption that it
> was complete. Looking into it using the ros-pull-request-builder It appears
> that @smits <https://github.com/smits> has not added write access. And in
> reviewing the instructions now I saw that adding Write access was buried in
> the paragraph header, and not in the enumerated tasks in the "alternate"
> approach. I've updated
> <http://wiki.ros.org/action/diff/buildfarm/Pull%20request%20testing?action=diff&rev1=18&rev2=19>
> the docs to make that clearer as I believe that it was not clearly enough
> documented if an experienced community member missed a step.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/ros/rosdistro/pull/15414#issuecomment-313903427>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AAG0m9q1TTHf9bHMvyHPpwrwJZZ9EURFks5sMHxjgaJpZM4OKUC1>
> .
>
-- 
Ruben Smits, Roboticist - Founder
+32 479 511 786
Intermodalics - Gaston Geenslaan 9, 3001 Heverlee - BELGIUM
www.intermodalics.eu

dirk-thomas commented:
@smits I have accepted the invitation for the buildfarm user. Therefore the PR integration should now work.

---

> Thus without any statement to the otherwise I took the assumption that it was complete.
> From the above comments @smits stated that he followed the instructions so I am taking the basic assumption that he has completed adding the write access. If you, who receive ros-pull-request-builders notifications, mentioned that you hadn't seen a request to grant access I would believe otherwise. But you didn't mention anything like that or otherwise indicate that you didn't think that ros-pull-request-builder had access in your comment.

I specifically raised the question since I haven't seen any invitation email. My comment with that questions was made four days before you merged the PR. With a question like this pending please don't merge future PR until the author had a chance to respond.

> Merging this doesn't transfer the responsibility for this. It is still the responsibility of the maintainer to validate that the PR is working as stated.

I will try eloborate why I think merging the PR changes the responsibility quite a bit. If nodody takes any action afterwards the problem is that the PR integration doesn't work and we "waste" the job resources. So it is on us to remove the job again if we don't want that to be the case forever. Otherwise with the PR not being merged we don't need to act on this until there is a comment or update on the pull request since no build farm resources are being used yet.
